components/toast/README.md

```diff
@@ -0,0 +1,168 @@
+  ```css
+  gameface-toast.custom-toast-class {
+      animation-name: customSlideUp;
+  }
+  @keyframes customSlideUp {
+      from { transform: translateY(100%); opacity: 0; }
+      to { transform: translateY(0); opacity: 1; }
+  }
+- **Fade Out Animation**: The default fade-out animation can be overridden by using the `guic-toast-hide` class along with a custom animation named `guic-toast-fade-out`. Ensure that any custom animations maintain the same name (`guic-toast-fade-out`) as the component’s `hide` method is specifically designed to listen for this animation's end event. To effectively override the default, your custom CSS rules must have a specificity greater than `0,1,1`. Here’s how you could define a custom fade-out animation:
+  ```css
+  .custom-toast-class.guic-toast-hide {
+      animation-name: guic-toast-fade-out; /* Use the same name for compatibility */
+      animation-duration: 0.8s; /* Custom duration */
+  }
+  @keyframes guic-toast-fade-out {
+      from { opacity: 1; }
+      to { opacity: 0; }
+  }
+  ```
+
+Specifying the content
+=========================
+
+Use the `message` slot to specify the message of the toast.
+
+~~~~{.html}
+<gameface-toast gravity="top" position="left">
+    <div slot="message">Toast on top left</div>
+</gameface-toast>
+~~~~
+
+Use the `close-btn` slot to include a close button in your toast:
+
+~~~~{.html}
+<gameface-toast class="styled-toast" gravity="top" position="right" timeout="5000">
+  <div slot="message">I have a target element</div>
+  <div slot="close-btn">x</div>
+</gameface-toast>
+~~~~
+
+Keep in mind that you **must** provide some content inside the slot, otherwise the close functionality of the button won't work. That's because the slot expects some content inside and then attaches the closing logic. If you don't specify a `close-btn` slot the toast will require manual removal (if timeout attribute is missing).
+
+
+Full list of the available attributes can be found [here](#Attributes).
+
+## How to use
+
+To use the toast component you must first add the element in your html:
+
+```{.html}
+<gameface-toast gravity="top" position="right" timeout="5000">
+  <div slot="message">I am a positioned toast!</div>
+  <div slot="close-btn">x</div>
+</gameface-toast>
+```
+### Showing the toast
+
+To display the toast you have 2 options:
+1. Manually getting the toast with js and calling the toast method `show()`
+2. Specifying a target element with the `target` attribute with an element selector as the value of the attribute. When the target element is clicked the `show()` method will be called on the toast.
+
+After the first call of the `show()` method 6 div containers will be created to host the toasts. The initialization of the containers will happen only the first time an instance of the toast component has been shown. These containers specify each place the toast can appear on and are implemented to allow for an easier way to stack toasts.
```

Comment:
```suggestion
After the first call of the `show()` method six containers will be created to host the toasts. The initialization of the containers will happen only the first time an instance of the toast component has been shown. These containers specify each place the toast can appear on and are implemented to allow for an easier way to stack toasts.
```

I don't care about if there are divs or not :D IMO we can omit `div` word here.

Reply:
true ;d, fixed